packages/mobile/src/components/action-drawer/ActionDrawer.tsx

```diff
@@ -114,17 +123,24 @@ export const ActionDrawerContent = (props: ActionDrawerContentProps) => {
 // `ActionDrawer` is a drawer that presents a list of clickable rows with text
 const ActionDrawer = (props: ActionDrawerProps) => {
   const {
-    modalName,
     rows,
     styles: stylesProp,
     disableAutoClose,
     children,
     ...other
   } = props
-  const { onClose } = useDrawerState(modalName)
+  const { onClose } =
```

Comment:
what's going on here? why do we need to support both exactly?

Reply:
yeah super weird, but there is "app-drawer" which are the common modals, and "native-drawer" which is the mobile specific drawers. By enabling both, we can use ActionDrawer in the mobile drawer context, which is the unlock to use it for messages.